viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -289,7 +309,7 @@ void CartoFacade::IOInit() {
     }
 
     // TODO: google cartographer will termiante the program if
-    // the internal state is invalid
```

Comment:
Hmm, are you sure this should say carto facade? I thought this was talking about google cartographer's internal state

Reply:
fixed